examples/dflowfm_local/dflowfm/crsloc.ini

```diff
@@ -10,333 +10,291 @@ branchId     = river_2                       # Branch on which the cross section
 chainage     = 517.667                       # Chainage on the branch (m).
 shift        = -10.0                         # Vertical shift of the cross section definition [m]. Defined positive upwards.
 definitionId = rect_h10.000_w50.000_c0_point # Id of cross section definition.
-locationtype = 1d
```

Comment:
Is this change because of hydrolib-core update or somehow a new version of Delft3d-FM? Just curious to understand.

Reply:
This field never existed but in the past HYDROLIB-core wrote it anyway, it was ignored by delft3dfm. Since 0.8.0 it has been removed from HYDROLIB-core so it can also not be present in this file anymore. More info in https://github.com/Deltares/HYDROLIB-core/pull/683